<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_encrypt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>620</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>encrypt</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_cipher_text">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>481</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Plantext</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dang Minh Kha - 2280601353</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_decrypt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>620</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>decrypt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cipher text</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_key">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RAILFENCE CIPHER </string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_plain_text">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>481</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
